### PR TITLE
Http logging

### PR DESCRIPTION
**Pull request by smebberson**
Provide a brief summary of your PR...

## Tasks

- Link to the task in notion.

## Related PRs

- Link to the related PR.

## Notes

- Any additional notes you think will be useful.

## Verification and testing

**If there is both a problem and a solution, these steps should document how to demonstrate the problem that is being solved. If there is no problem (it's just a feature) you can remove the _Recreate_ section below.**

### Verification

These are the steps to demonstrate the problem being solved:

- [ ] A list of steps to demonstrate the problem.

### Testing

These are the steps to demonstrate the solution being solved; they typically include steps to update the repository to a new state containing the code that fixes the problem.

- [ ] A list of steps to demonstrate the problem.

## Deployment

- [ ] Post deployment steps (scripts to be run) and testing.
- [ ] Verify all [testing](../tree/master/docs/testing.md) steps.

### CODE REVIEW

File: packages/log-structured/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "scripts": {
+        "test": "jest"
+    },
+    "dependencies": {
+        "pump": "3.0.0"
+    },
+    "devDependencies": {
+        "pino-http": "5.5.0",
+        "sink": "0.1.1"
```
Comment (allanchau) [inline packages/log-structured/package.json:19]: Is this right? Shouldn't they just be normal dependencies?

File: packages/log-structured/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const parse = (input) => {
+    try {
+        return JSON.parse(input);
+    } catch (e) {
+        return input;
+    }
+};
+
+module.exports = () => {
+    return new Transform({
```
Comment (allanchau) [inline packages/log-structured/index.js:46]: Do you need the extra `return` here? Couldn't you just return it directly?

**Review by allanchau — CHANGES_REQUESTED** [review]
@smebberson Just a few minor things

File: packages/log-structured/package.json (hunk 1 above)
Comment (smebberson) [reply packages/log-structured/package.json:19]: @allanchau, that only used in the tests...

File: packages/log-structured/index.js (hunk 2 above)
Comment (smebberson) [reply packages/log-structured/index.js:46]: Whoops, I forgot to change that back.

File: packages/log-structured/package.json (hunk 1 above)
Comment (allanchau) [reply packages/log-structured/package.json:19]: @smebberson Oh sorry wrong file, I was suppose to make the comment in packages/log-http/package.json. `pino-http` at least should be a dependency?

**Review by allanchau — APPROVED** [review]

**Pull request merged**